scripts/backup.sh

```diff
@@ -25,15 +25,18 @@
 #   sudo bash ./scripts/backup.sh 2 pi
 #     This will only produce a backup in the rollowing folder and change all the permissions to the 'pi' user.
 
-if [ -d "./menu.sh" ]; then
-	echo "./menu.sh file was not found. Ensure that you are running this from IOTstack's directory."
-  exit 1
+if [ ! -f "menu.sh" ]; then
+  cd "$HOME/IOTstack"
```

Comment:
`cd "$(dirname $(readlink -f $0))/.."` Does this work for you instead? Trying not to lock IOTstack to `~/IOTstack`

Reply:
Sorry for taking some time to reply @Slyke . It's of course a lot better to avoid hardcoding the IOTstack directory. I did something similar that I think should also work in the unlikely case that the path contains spaces.